apps/datahub/src/app/record/custom-api/custom-api.component.html

```diff
@@ -0,0 +1,47 @@
+<div
+  class="bg-primary-opacity-10 pt-8 overflow-visible sm:pt-5"
+  style="height: 512px"
+>
+  <div class="text-[28px]" translate>Générer une URL personnalisée</div>
+  <div
+    class="flex flex-col bg-white container-lg px-6 py-5 lg:mx-auto ng-star-inserted card-shadow"
+  >
+    <div class="flex flex-row">
+      <div class="text-[23px]" translate>Créer votre requête</div>
+      <button
+        class="bg-primary-opacity-25 inline-flex items-center justify-center px-2 py-1 text-13 font-medium leading-none text-white rounded capitalize text-primary-lightest group-hover:bg-primary transition-colors"
+      >
+        REINITIALISER
+      </button>
+    </div>
+    <div class="flex flex-row justify-around">
+      <div class="flex flex-col">
+        <p>Nombre d'enregistrements</p>
+        <gn-ui-text-input [value]="20" [required]="true" hint="Limit">
+        </gn-ui-text-input>
+      </div>
+      <div class="flex flex-col">
+        <p>Numéro du 1er enregistrement</p>
+        <gn-ui-text-input [value]="20" [required]="true" hint="Offset">
+        </gn-ui-text-input>
+      </div>
+      <div class="flex flex-col">
+        <p>Intervalle de temps</p>
+      </div>
+    </div>
+  </div>
+  <div class="text-[23px]" translate>URL personnalisée</div>
+  <gn-ui-text-input [value]="'Your URL here'" [required]="false" hint="">
+  </gn-ui-text-input>
+  <div class="text-[23px]" translate>URL racine</div>
```

Comment:
Does not seem optimzed to switch from 23 to 20 like this.
BTW, do we have a zeplin or figma ?

Reply:
There is a figma, I got the link to it after committing everything I've done here. I will use it to have better styling values from now on. Styling is not done/approximative right now, I'm focusing on the functionalities first.